docs/DeveloperGuide.md

```diff
@@ -452,6 +447,145 @@ facilitates the lending of money and ensures data integrity by validating input
 The TransactionList class provides essential functionalities for managing a list of transactions.
 Its methods facilitate the addition, removal, editing, and retrieval of transactions, ensuring efficient management of transactions within a group.
 
+### DateTime
```

Comment:
Add DateTime to the ToC for D&I (look right under design and implementation)

Reply:
Added!

Reply:
![image](https://github.com/AY2324S2-CS2113-T15-1/tp/assets/13176823/6c1e5d65-0e97-4743-b898-409461a63044)
I meant here